fix: MD033/no-inline-html

nschonni opened the pull request:
Latest markdownlint changed the regex and is flagging some things that aren't actually fully swallowed elements anymore. Still escaped the elements
DbLimitExpession -> DbLimitExpression

nschonni committed:
fix: MD030/list-marker-space

Spaces after list markers

nschonni committed:
fix: MD047/single-trailing-newline

Files should end with a single newline character

rpetrusha commented on docs/framework/unmanaged-api/metadata/corelementtype-enumeration.md:
```diff
@@ -1,144 +1,147 @@
-  
-|Type modifier|Format|  
-|-------------------|------------|  
-|`ELEMENT_TYPE_PTR`|ELEMENT_TYPE_PTR <a `CorElementType` value>|  
```
The change from "a" to "an" here is incorrect. Because m is pronounced as "em", with a leading vowel sound, "an" should be used.

nschonni replied:
Not sure if i understood the rule, but it's fixed 😄

rpetrusha replied:
Thanks, @nschonni. I'll merge your PR now.

rpetrusha reviewed (changes requested):
Your changes look good, @nschonni, except for the replacement of "an" with "a".

nschonni committed:
fix: MD033/no-inline-html

Latest markdownlint changed the regex and is flagging some things that aren't actually fully swallowed elements anymore. Still escaped the elements
DbLimitExpession -> DbLimitExpression

rpetrusha commented:
Actually, I can't merge, @nschonni. There's a merge conflict, and for some reason I can't open the web editor to resolve it. Can you resolve it?

nschonni committed:
fix: MD012/no-multiple-blanks

Multiple consecutive blank lines

nschonni commented:
@rpetrusha i've rebased it for the merge conflict

rpetrusha commented:
Thanks, @nschonni. I'll merge now.
